amplify/backend/api/householdapp/schema.graphql

```diff
@@ -111,7 +111,7 @@ type Event
 {
   id: ID!
   date: AWSDateTime
-  eventHandlerId: ID! @index(name: "byEventHandler")
+  eventHandlerId: ID! @index(name: "byEventHandler", sortKeyFields: ["id"])
```

Comment:
adding this `sortKey` might have generated some weird queries, check next comment